### PR TITLE
connect to #159 and connect to #207 style campaign page

### PR DESCRIPTION
**Pull request by ximsce**

### CODE REVIEW

File: client/src/components/SignatureList.js [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+  }
+
+  render() {
+    const { signatures } = this.props;
+    return (
+    <div className="signature-wrapper">
+      <h2 className={this.state.areSignaturesExpanded ? ("signature-list-down") : "signature-list-up"} onClick={this.toggleSignatures}>View Signatures</h2>
+      <ul className={this.state.areSignaturesExpanded ? ("signature-list-show") : "signature-list-hide"}>{signatures && signatures.map(({id, name}) => <li key={id}>{name}</li>)}</ul>
+    </div>
+    );
```
Comment (chompasina) [inline client/src/components/SignatureList.js]: Finally some better styling! Nice work.

File: client/src/stylesheets/components/_SignCampaign.scss [hunk 2]
```diff
@@ -41,3 +47,45 @@ h2 {
   box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15) inset, 0 0 6px rgba(0, 0, 0, 0.2) inset;
   border-color: #000\9;
 }
+
+.signature-list-show {
```
Comment (chompasina) [inline client/src/stylesheets/components/_SignCampaign.scss]: do we need this class listed?

File: client/src/stylesheets/components/_SignCampaign.scss [hunk 3]
```diff
@@ -41,3 +47,45 @@ h2 {
...
+.signature-list-down:after {
+  content:'';
+  position: absolute;
+  top: 100%;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  width: 0;
+  height: 0;
+  border-bottom: solid 5px black;
```
Comment (chompasina) [inline client/src/stylesheets/components/_SignCampaign.scss]: Do we want these to be responsive values or will it not matter once we have breakpoints? 🤔

Comment (ximsce) [reply client/src/stylesheets/components/_SignCampaign.scss]: I think these are temporary values until we can do something better, so yes... soon.

**Pull request merged**